crates/wasm-mutate/src/lib.rs

```diff
@@ -144,9 +144,8 @@ impl WasmMutate {
         self.raw_mutate_func = raw_mutate_func;
         self
     }
-
-    /// Run this configured `WasmMutate` on the given input Wasm.
-    pub fn run<'a>(&self, input_wasm: &'a [u8]) -> Result<Vec<u8>> {
+    /// Run this configured `WasmMutate` on the given input Wasm by providing a random number generator
+    pub fn run_with_rng<'a>(&self, input_wasm: &'a [u8], rng: &mut SmallRng) -> Result<Vec<u8>> {
```

Comment:
Is this method necessary? Can we use the `wasm-mutate-stats`'s RNG to generate a new seed for each of the `WasmMutate` instances it creates instead of making `wasm-mutate`'s API bigger?